sklearn/ensemble/_forest.py

```diff
@@ -420,8 +421,14 @@ def fit(self, X, y, sample_weight=None):
                     "RandomForestRegressors and ExtraTreesRegressors.",
                     FutureWarning,
                 )
-
+        # TODO(1.3): Remove "entropy" and "auto"
         elif isinstance(self, (RandomForestClassifier, ExtraTreesClassifier)):
+            if self.criterion == "entropy":
+                warn(
+                    "Criterion 'entropy' was deprecated in v1.1 and will be removed "
```

Comment:
From the discussions, it seems like we want to keep both names and not deprecate entropy?

Reply:
right.

Reply:
I missed that one.